### ISSUE TITLE
Object (interface) not found when bulk importing IP Addresses

### ISSUE THREAD

**Issue by thiagomarafeli** [netbox-community/netbox#15034]
### Deployment Type

Self-hosted

### NetBox Version

v3.7.1

### Python Version

3.10

### Steps to Reproduce

1. Go to Virtualization > Interface and create a new interface
2. Go to IPAM > IP Addresses and go to the bulk import screen
3. Try to import a CSV with an IP referencing that interface

Example of import, assuming an interface created in step 1 with the name "eni-123456789", and given the fact that the accessor  is `name`:
```csv
address,status,interface
12.123.123.123/32,active,eni-123456789
```

In this case, I'm trying to associate the IP with an interface attached to a VM and a VRF. Although it's possible to correctly select the interface over UI, on bulk import this is not possible.

### Expected Behavior

A successful import

### Observed Behavior

An error showed up:
`Record 1 interface: Object not found: eni-123456789`

**Comment by thiagomarafeli** [comment]
I myself realized that it was also necessary to inform in this import the `virtual_machine` which the interface was associated with.
After that, it worked.

But, I only realized that issue after I tried to set as the primary IP for that interface and an error showed up saying that it was necessary to also inform the device or VM to work. It would be nice if that same message shows when the user try to import the interface without the parent.

Also... isn't missing the FHRP Group in bulk import? I never used FHRP, but if there is the `device` and `virtual_machine` fields, shouldn't have a `fhrp_group` or something like that in bulk import?

I will leave this issue open because of these 2 last comments, but, about the original question, it's not a bug and my issue doesn't exist anymore.
